pkg/network/netlink/conntracker.go

```diff
@@ -72,7 +72,9 @@ func NewConntracker(procRoot string, maxStateSize, targetRateLimit int, listenAl
 	done := make(chan struct{})
 
 	go func() {
+		start := time.Now()
 		conntracker, err = newConntrackerOnce(procRoot, maxStateSize, targetRateLimit, listenAllNamespaces)
+		log.Infof("conntrack initialization took %s", time.Now().Sub(start))
```

Comment:
I don't think we need to log this, at least not at info. We already log an error if conntrack does not get enabled. We can assume that conntrack starts if the system-probe starts.